src/test/unit/objects/test_virtual_file_path.py

```diff
@@ -1,27 +1,25 @@
-import unittest
-from test.common_helper import create_test_firmware, create_test_file_object
+from helperFunctions.virtual_file_path import join_virtual_path
+from test.common_helper import create_test_file_object, create_test_firmware
 
 
-class TestVirtualFilePath(unittest.TestCase):
+def test_add_file_object():
```

Comment:
Ugh

Reply:
Thoughts:
- No test for child two
- This looks like three tests (root, first layer, second layer) even if that duplicates code

Reply:
split into three tests